tensorflow/python/estimator/estimator.py

```diff
@@ -259,8 +258,16 @@ def get_variable_value(self, name):
     Raises:
       ValueError: If the Estimator has not produced a checkpoint yet.
     """
+    def _check_string_or_not(name):
+        if isinstance(name, six.string_types):
+            return name
+        raise TypeError("Received type {} and was expecting an input of type string or a list of strings".format(type(name)))
+
     _check_checkpoint_available(self.model_dir)
-    return training.load_variable(self.model_dir, name)
+    if isinstance(name, six.string_types):
+        return training.load_variable(self.model_dir, name)
+    else:
+        return {v:traning.load_variable(self.model_dir, _check_string_or_not(v)) for v in name}
```

Comment:
Still need to fix the code here. And I expect you also need to fix spacing -- the linter is pretty unforgiving.

Reply:
Yup, Sorry I thought you guys used 8 spacing instead of customized. I just fixed them. Would you review it, please?